TerminatorPlus-API/src/main/java/net/nuggetmc/tplus/api/agent/legacyagent/ai/IntelligenceAgent.java

```diff
@@ -100,22 +100,22 @@ private void task() throws InterruptedException {
     private void runGeneration() throws InterruptedException {
         generation++;
 
-        print("Starting generation " + ChatColor.RED + generation + ChatColor.RESET + "...");
+        print("Starting generation " + NamedTextColor.RED + generation + NamedTextColor.WHITE + "...");
```

Comment:
I don't think this would work. Could you please use either ChatColor or convert it to components?
Please correct me if I'm wrong.

Reply:
ChatColor is deprecated: https://search.app?link=https%3A%2F%2Fjd.papermc.io%2Ffolia%2F1.20%2Forg%2Fbukkit%2FChatColor.html&utm_campaign=aga&utm_source=agsadl2%2Csh%2Fx%2Fgs%2Fm2%2F4

Reply:
Yes I am aware, which is why I'm suggesting we use components/minimessage

Reply:
Moved it to MiniMessage